contracts/interface/IPolicyManager.sol

```diff
@@ -42,6 +42,7 @@ interface IPolicyManager is IERC721Enhanced {
         uint40 expirationBlock;
         uint24 price;
         bytes positionDescription;
+        address riskStrategy;
```

Comment:
it might be more efficient to move riskStrategy before positionDescription, both here and in related function calls

Reply:
imo, this is also good task for the new team members.